src/app/directives/itemDirective.js

```diff
@@ -0,0 +1,11 @@
+export function itemDirective () {
+	return {
+		restrict: 'E',
+		scope: {
+			list: '=', 
```

Comment:
is it really needed to create scope vars with two way data binding? Could we use @ single binding instead? Remember the more two way data binding scope vars, the less performant becomes the app. This comment apply for showYear and type too.